index.html

```diff
@@ -3413,37 +3413,37 @@ <h3>Syntactic Sugar</h3>
         <h3>Media Types</h3>
 
         <p>
-Media types as defined in [[RFC6838]] serve a useful purpose with verifiable
-credentials, especially regarding identification of a particular syntax that is
-in use with a verifiable credential.
+Media types, as defined in [[RFC6838]], identify the syntax used to express a
+<a>verifiable credential</a> as well as other useful processing guidelines.
         </p>
         <p>
-Syntaxes SHOULD be identified by a media type, and certain conventions as
-outlined in this section SHOULD be followed when defining or using media types
-with verifiable credentials.
+Syntaxes used to express the data model in this specification SHOULD be
+identified by a media type, and conventions outlined in this section SHOULD be
+followed when defining or using media types with <a>verifiable credentials</a>.
         </p>
         <p>
-Any media types associated with the core data model are listed in the section on
-<a href="#iana-considerations">IANA Considerations</a> for registration with
-IANA.
+Any media types associated with the core data model are listed in the Section
+<a href="#iana-considerations"></a>.
         </p>
         <p class="issue" data-number="1060">
-Presence of the term "verifiable" (or a shortened version) in the media type may
-or may not imply a proof.  Original proposed language is tracked in the issue
-corresponding to this note.
+The VCWG is currently discussing whether the presence of the term "verifiable"
+(or a shortened version) in the media type may or may not imply a proof. The
+Working Group is seeking implementation experience related to the use of media
+types and the expectation related to whether or not a particular media type
+is expected to be secure with a particular proof.
         </p>
         <p>
-Use of the term `credential` in a media type related to a syntax of verifiable
-credentials <i>without</i> the corresponding use of `verifiable` as described
-above <i>does not</i> imply the presence of a proof with the
-<a href="#dfn-credential">credential</a>.
+Use of the term "credential" in a media type related to a syntax of
+<a>verifiable credentials</a> <em>without</em> the corresponding use of
+"verifiable" as described above <em>does not</em> imply the presence of a
+proof with the <a>credential</a>.
```

Comment:
I think we should apply the same philosophy to all media types related to this specification that media types describe a secured 'thing' and media type itself does not imply the proofing mechanism. given the base media type is application/vc+ld+json; all extensions of VCDM core spec should use `vc+`.

happy to file a separate issue, since I think this PR is meant to be editorial...